src/tools.json

```diff
@@ -59,6 +59,43 @@
             }
         }
     },
+    "func": {
+        "description": "Function CLI tool",
+        "vendor": "Red Hat, Inc.",
+        "name": "func",
+        "version": "1.10.0",
```

Comment:
@msivasubramaniaan Bump the version to latest func 1.10.1 https://github.com/knative/func/releases/tag/knative-v1.10.1